lib/validators/mbtiles.js

```diff
@@ -77,9 +77,13 @@ function deprecatedCarmen(db, callback) {
 
 function zoomCheck(db, limits, callback) {
   function getMaxZoom(type, cb) {
-    var query = type === 'tiles' ?
-      'SELECT MAX(zoom_level) as z FROM tiles' :
-      'SELECT MAX(zoom_level) as z FROM grids' ;
+    var query =
```

Comment:
Not really a problem, but why this instead of a good-old switch?

Reply:
No good reason. I'll change it. It was a ternary expression before, so I just didn't change the form.